Shared/Features/Freaks/ViewModel/FreakDetailsViewModel.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+final class FreakDetailsViewModel {
+    private var freak: Freak
+
+    init(freak: Freak) {
+        self.freak = freak
+    }
+
+    var imageName: String {
```

Comment:
`imageName` seems to be added here but not used. We should avoid adding unnecessary code to our codebase.